lib/platform/bitbucket-server/index.spec.ts

```diff
@@ -1,3 +1,5 @@
+// TODO fix mocks
+/* eslint-disable jest/expect-expect */
```

Comment:
```suggestion
/* eslint jest/expect-expect: 1 */
```
Should be fixed, warn for now